support/modules-artifact-gen/directory-artifact-gen

```diff
@@ -7,17 +7,19 @@ show_help() {
 
 Simple tool to generate Mender Artifact suitable for directory Update Module
 
-Usage: $0 [options] directory
+Usage: $0 [options] directory [-- [options-for-mender-artifact] ]
```

Comment:
This is fine.

However, makes me wonder if we should do this even simpler. The generator could only parse the parameters required (only `dest-dir`) and just pass the rest to the tool, so that it will fail there is something mandatory is missing (`artifact-name` or `device-type` for example).

Reply:
Yes, that is an option, but then the user has to pass those arguments after the `--` which is perhaps a bit more user unfriendly? Especially given that they are required options. I don't think we should pass all unknown arguments to the tool, because it makes it risky to add arguments to the gen script later (namespace overlap). Does that make sense?

Reply:
Makes sense. Specially the last bit which I didn't think about.

Good.